tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "esModuleInterop": true,
+    "target": "es2015",
+    "downlevelIteration": true,
+    "importHelpers": true,
+    "sourceMap": true,
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "allowJs": false,
+    "checkJs": false,
+    "resolveJsonModule": true,
+    "lib": ["dom", "es2015", "es2016", "es2017", "esnext"],
+    "forceConsistentCasingInFileNames": true,
+    "experimentalDecorators": true,
+    "noImplicitAny": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noImplicitThis": true,
+    "noUnusedLocals": true,
+    "skipLibCheck": true,
+    "strictNullChecks": true,
+    "suppressImplicitAnyIndexErrors": true,
+    "types": ["jest", "node"],
+    "baseUrl": "./src/",
+    "outDir": "./build"
+  },
+  "include": ["src/**/*.ts", "test/**/*.ts", "node_modules/web3-*/types/*.d.ts", "node_modules/moment/moment.d.ts"],
```

Comment:
Do we really need `"node_modules/web3-*/types/*.d.ts", "node_modules/moment/moment.d.ts"`?
These should be picked up automatically
Let's try without and add back if necessary.